core/src/main/java/tc/oc/pgm/PGMPlugin.java

```diff
@@ -228,6 +223,9 @@ public void onEnable() {
       asyncExecutorService.scheduleAtFixedRate(new ShouldRestartTask(), 0, 1, TimeUnit.MINUTES);
     }
 
+    // Set name provider
+    PlayerComponentProvider.PROVIDER.set(new NameComponentProviderImpl());
```

Comment:
The whole of name rendering is absurdly messy in community branch, that needs to be sorted out. Needs proper `PlayerComponent`s which render to per-player view, but not via a text-replace